applications/main/nfc/plugins/supported_cards/social_moscow.c

```diff
@@ -0,0 +1,254 @@
+    {.a = 0x8765b17968a2, .b = 0x6202a38f69e2},
+    {.a = 0x40ead80721ce, .b = 0x100533b89331},
+    {.a = 0x0db5e6523f7c, .b = 0x653a87594079},
+    {.a = 0x51119dae5216, .b = 0xd8a274b2e026},
+    {.a = 0x51119dae5216, .b = 0xd8a274b2e026},
+    {.a = 0x51119dae5216, .b = 0xd8a274b2e026},
+    {.a = 0x2aba9519f574, .b = 0xcb9a1f2d7368},
+    {.a = 0x84fd7f7a12b6, .b = 0xc7c0adb3284f},
+    {.a = 0xa0a1a2a3a4a5, .b = 0x7de02a7f6025}};
+
+static const MfClassicKeyPair social_moscow_4k_keys[] = {
+    {.a = 0xa0a1a2a3a4a5, .b = 0x7de02a7f6025}, //1
+    {.a = 0x2735fc181807, .b = 0xbf23a53c1f63}, //2
+    {.a = 0x2aba9519f574, .b = 0xcb9a1f2d7368}, //3
+    {.a = 0x84fd7f7a12b6, .b = 0xc7c0adb3284f}, //4
+    {.a = 0x73068f118c13, .b = 0x2b7f3253fac5}, //5
+    {.a = 0x186d8c4b93f9, .b = 0x9f131d8c2057}, //6
+    {.a = 0x3a4bba8adaf0, .b = 0x67362d90f973}, //7
+    {.a = 0x8765b17968a2, .b = 0x6202a38f69e2}, //8
+    {.a = 0x40ead80721ce, .b = 0x100533b89331}, //9
+    {.a = 0x0db5e6523f7c, .b = 0x653a87594079}, //10
+    {.a = 0x51119dae5216, .b = 0xd8a274b2e026}, //11
+    {.a = 0x51119dae5216, .b = 0xd8a274b2e026}, //12
+    {.a = 0x51119dae5216, .b = 0xd8a274b2e026}, //13
+    {.a = 0xa0a1a2a3a4a5, .b = 0x7de02a7f6025}, //14
+    {.a = 0xa0a1a2a3a4a5, .b = 0x7de02a7f6025}, //15
+    {.a = 0xa0a1a2a3a4a5, .b = 0x7de02a7f6025}, //16
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //17
+    {.a = 0x2aba9519f574, .b = 0xcb9a1f2d7368}, //18
+    {.a = 0x84fd7f7a12b6, .b = 0xc7c0adb3284f}, //19
+    {.a = 0x2aba9519f574, .b = 0xcb9a1f2d7368}, //20
+    {.a = 0x84fd7f7a12b6, .b = 0xc7c0adb3284f}, //21
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //22
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //23
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //24
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //25
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //26
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //27
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //28
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //29
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //30
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //31
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //32
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //33
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //34
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //35
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //36
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //37
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //38
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //39
+    {.a = 0xa229e68ad9e5, .b = 0x49c2b5296ef4}, //40
+};
+
+static bool social_moscow_get_card_config(SocialMoscowCardConfig* config, MfClassicType type) {
+    bool success = true;
+    if(type == MfClassicType1k) {
+        config->data_sector = 15;
+        config->keys = social_moscow_1k_keys;
+    } else if(type == MfClassicType4k) {
+        config->data_sector = 15;
```

Comment:
if I see it correctly, this will use key `0xa0a1a2a3a4a5` which introduce the same false-positive problem that I just fixed for 1k-cards in https://github.com/flipperdevices/flipperzero-firmware/pull/3467 for the wash-city parser:
the 0xa0.. key is very commonly used and if any plugins verify func uses only that key, chances are high for false-positive , essentially breaking reading of alot cards

Reply:
I chose block 15 for verification as it contains the basic data of the social card, I agree that both keys A and B need to be checked at once.